algorithmic_efficiency/workloads/imagenet_resnet/imagenet_pytorch/workload.py

```diff
@@ -88,10 +100,10 @@ def _build_dataset(self,
       else:
         sampler = data_utils.DistributedEvalSampler(
             dataset, num_replicas=N_GPUS, rank=RANK, shuffle=False)
-      batch_size //= N_GPUS
+      per_device_batch_size = global_batch_size // N_GPUS
     dataloader = torch.utils.data.DataLoader(
         dataset,
-        batch_size=batch_size,
+        batch_size=per_device_batch_size,
```

Comment:
`per_device_batch_size` is not defined if `USE_PYTORCH_DDP` is False.

Reply:
ah good catch, thanks!